source/kubernetes/user-access.rst

```diff
@@ -46,6 +46,110 @@ assigned to their account.
 |               | admin namespace.                                                 |
 +---------------+------------------------------------------------------------------+
 
+More information
+----------------
+
+.. raw:: html
+
+   <details>
+   <summary>For a more comprehensive list of the exact permissions that each
+   role gives, you can <a>click here</a> to view a full list for the kubernetes
+   roles.</summary>
+
+.. code-block:: console
+
+   +----------------------+-----------------------------------------------------+
+   | Role                 | Permissions                                         |
+   +======================+=====================================================+
+   | k8s_admin            | container.*                                         |
```

Comment:
@danielobyrne May I know where did you get the source to generate this form? Because it doesn't match what we have, see https://github.com/openstack/magnum/blob/master/etc/magnum/keystone_auth_default_policy.sample

Reply:
At the start of these I can't remember who but they directed me here and mentioned that our roles were largely the same: https://cloud.google.com/iam/docs/understanding-roles#kubernetes-engine-roles